presto-docs/src/main/sphinx/release/release-0.181.rst

```diff
@@ -0,0 +1,68 @@
+=============
+Release 0.181
+=============
+
+General Changes
+---------------
+
+* Fix query failure and untracked memory usage when query contains
```

Comment:
`untracked memory usage` sounds weird. How about `Fix query failure and memory usage tracking ...`?